core/src/main/java/smashdudes/graphics/effects/Particle.java

```diff
@@ -38,4 +42,19 @@ public class Particle
     public float gEnd;
     public float bEnd;
     public float aEnd;
+
+    public void render(SpriteBatch sb)
+    {
+        float t = 1.0f  - life / initialLife;
+
+        float scale = MathUtils.lerp(scaleStart, scaleEnd, t);
+
+        float r = MathUtils.lerp(rStart, rEnd, t);
+        float g = MathUtils.lerp(gStart, gEnd, t);
+        float b = MathUtils.lerp(bStart, bEnd, t);
+        float a = MathUtils.lerp(aStart, aEnd, t);
+
+        sb.setColor(r, g, b, a);
+        sb.draw(RenderResources.getTexture("textures/particleTexture.png"), x - scale / 2, y - scale / 2, scale, scale);
+    }
```

Comment:
I feel this should be within the emitter, as the emitter will have to select the particle texture. 

Another thing - do we want to support multiple textures per emitter? I.E an array where we select a random one? Maybe out of scope

Reply:
This was why I was thinking we would have this done per particle, since then you would give the particle it's desired texture for rendering via it's creation

Reply:
The same could be said for the updating. 
This does add a dependency on spritebatch to the particle class, but it doesn't really matter where the code goes (emitter vs particle class) and is preference.

Reply:
I actually think this is even more a case for the rendering to be in the emitter. 
When we submit to the spritebatch we want draws to be grouped by texture, there is not way to to that within the particle itself.